### ISSUE TITLE
Doesn't download some files

### ISSUE THREAD

**Issue by Resermag** [SidhNor/SITracker#61]
After downloading some files have empty body block. Example : http://samlib.ru/editors/a/aleksej/zemlimechaimagiiwoploshennyjwkamne.shtml

**Comment by SidhNor** [comment]
Please add the author without the /editors part in the link.
I tried adding the author as:
http://samlib.ru/a/aleksej

And then downloaded the zemlimechaimagiiwoploshennyjwkamne.shtml file and it wasn't blank.

Could you try it without editors?

**Comment by Resermag** [comment]
Yes, it works. Thanks for answer. Maybe will be good for adding testing search line for "editors". I doesn't know how much people have this problem too, but it will be good for adding some instrumets for founding this problem.
